.github/CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 # These are the default owners for the code and will
 # be requested for review when someone opens a pull request.
 
-* @abhi16394 @bharathsreekanth @JacobGros @mjsdell @jooseppi-luna @ashleyvjoy @Deepak-Ghivari @khareRajshree @KshitijaKakde @prabhu-dell @VamsiSiddu-7 @mbasha-dell @kumarkgosa
+* @abhi16394 @bharathsreekanth @JacobGros @mjsdell @jooseppi-luna @ashleyvjoy @Deepak-Ghivari @khareRajshree @KshitijaKakde @prabhu-dell @VamsiSiddu-7 @mbasha-dell @kumarkgosa @AkshaySainiDell @rishabhatdell @sakshi-garg1 @adarsh-dell @suryagupta4
```

Comment:
Can we have the code owners name arranged alphabetically, like it was originally added. Thanks.

Reply:
Done @khareRajshree